labelingtool/README.md

```diff
@@ -0,0 +1,20 @@
+# dida Labeling Tool
+
+This repository contains the code to run a simple labeling tool for image classification in a Jupyter notebook.
+
+<!-- It accompanies [this article](ADD-LINK) in the dida blog. -->
+
+## Requirements & Installation
+The following are required to run the tool:
+- Python 3
+- IPython
```

Comment:
Jupyter already depends on this, no? So we don't need to mention it.

Reply:
you are right, I will remove it